api/v1/db/oracledb/LearningDB.sql

```diff
@@ -0,0 +1,192 @@
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (28, 2, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (29, 3, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (30, 4, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (31, 5, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (32, 6, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (33, 7, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (34, 8, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (35, 9, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (36, 10, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (37, 11, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (38, 12, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (39, 13, 3);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (40, 1, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (41, 2, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (42, 3, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (43, 4, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (44, 5, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (45, 6, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (46, 7, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (47, 8, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (48, 9, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (49, 10, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (50, 11, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (51, 12, 4);
+INSERT INTO CARDS (CARD_ID, CARD_NUMBER_ID, SUIT_ID) VALUES (52, 13, 4);
+
+CREATE TABLE ROUNDS (
+    ROUND_ID NUMBER NOT NULL,
+    ROUND VARCHAR2(255 BYTE),
+    PRIMARY KEY (ROUND_ID)
+);
+
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES (1, 'blind');
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES (2, 'flop');
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES (3, 'turn');
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES (4, 'river');
+INSERT INTO ROUNDS (ROUND_ID, ROUND) VALUES (5, 'showdown');
+
+CREATE TABLE MEMBERS (
+    MEMBER_ID NUMBER NOT NULL,
+    MEMBER_NAME VARCHAR2(255 BYTE),
+    MEMBER_EMAIL VARCHAR2(255 BYTE),
+    MEMBER_LEVEL NUMBER NOT NULL,
+    MEMBER_EXP_OVER_LEVEL NUMBER NOT NULL,
+    MEMBER_PASSWORD VARCHAR2(255 BYTE),
+    PRIMARY KEY (MEMBER_ID),
+    UNIQUE (MEMBER_EMAIL)
+);
+
+CREATE TABLE GAMES (
+    GAME_ID NUMBER NOT NULL,
+    ROUND_ID NUMBER NOT NULL,
+    MINIMUM_BET NUMBER NOT NULL,
+    MAXIMUM_BET NUMBER NOT NULL,
+    BET_POOL NUMBER NOT NULL,
+    PRIMARY KEY (GAME_ID),
+    FOREIGN KEY (ROUND_ID) REFERENCES ROUNDS(ROUND_ID)
+);
+
+CREATE TABLE TABLE_CARDS (
+    OWNERSHIP_ID NUMBER NOT NULL,
+    GAME_ID NUMBER NOT NULL,
+    CARD_ID NUMBER NOT NULL,
+    PRIMARY KEY (OWNERSHIP_ID),
```

Comment:
For this table and other simple relationship tables that consist of two IDs, you shouldn't need to create a third ID to uniquely identify the relationship. Instead, you could have the primary key be a composite key of the two ids. For example here, you could have `GAME_ID` and `CARD_ID` together be a composite key for the table. This also prevents duplicate relationships.